libs/search/src/lib/elasticsearch/elasticsearch.service.ts

```diff
@@ -17,7 +17,15 @@ export class ElasticsearchService {
 
   buildPayload(state: SearchState): SearchParams {
     const { size, sortBy, filters } = state.params
-    const sort = sortBy ? [sortBy] : undefined
+    const sort: SortParams = sortBy
```

Comment:
Should be externalized cause it might be used to build another payload ?
Also, not tested yet